options/options.go

```diff
@@ -91,6 +96,7 @@ func New() (*Options, error) {
 	}
 
 	// TODO(options): Move this set-or-default logic to its own function.
+	opts.ScorecardOpts.Format = envVars.EnvInputResultsFormat
```

Comment:
```suggestion
	opts.ScorecardOpts.Format = envVars.EnvInputResultsFormat
```

`INPUT_` vars should be last resort.
Default to SARIF and only override if `INPUT_RESULTS_FORMAT` is set AND populated.